device/alpaka/src/utils/utils.hpp

```diff
@@ -0,0 +1,37 @@
+/** TRACCC library, part of the ACTS project (R&D line)
```

Comment:
Ahh, okay. I just see this new file. Yes, I like it. But is there a reason for keeping `definitions.hpp` alive? Why not move everything (needed) in here from there?

Reply:
Done this now....

The only reason I'd kept `definitions.hpp` around was as a top level place to define the various includes.
I think really, we will need to think up a sensible / clean way of doing that somewhere though.

(What I mean is conditional includes to include say `vecmem/copy`, but also `vecmem/cuda/copy` when compiled to a CUDA target, or `vecmem/hip/copy` for a HIP target and so on. Right now, the includes and any guards are duplicated in a few different places, which isn't ideal.)